test/NuGet.Core.Tests/NuGet.Frameworks.Test/FrameworkComparerTests.cs

```diff
@@ -55,6 +63,10 @@ public void FrameworkComparer_FrameworkOrderingWithPreferredAndNormalSorting()
             Assert.Equal(fw8, list[7]);
             Assert.Equal(fw9, list[8]);
             Assert.Equal(fw10, list[9]);
+            Assert.Equal(fw11, list[10]);
+            Assert.Equal(fw12, list[11]);
+            Assert.Equal(fw13, list[12]);
+            Assert.Equal(fw14, list[13]);
```

Comment:
Agreed. This case is not possible with the reducer so I added the split precedence lists (as we spoke about offline).